src/app/page.tsx

```diff
@@ -14,13 +14,9 @@ import { IconCheck } from "@tabler/icons-react";
 import React from "react";
 import { getLatestReleases } from "@/app/downloads/github";
 
-const InteractiveLogo = dynamic(() => import("../components/logo"), {
-  ssr: false,
-});
```

Comment:
Can we make this itself a client component? Will that make this unnecessary?

```ts
export default function Home() {
  const [latest, setLatest] = React.useState(null);
  const [loading, setLoading] = React.useState(true);

  React.useEffect(() => {
    const fetchReleases = async () => {
      try {
        const releases = await getLatestReleases();
        setLatest(releases.length > 0 ? releases[0] : null);
      } catch (err) {
        console.warn('Failed to fetch releases');
      } finally {
        setLoading(false);
      }
    };
    fetchReleases();
  }, []);
  ```

And render some loading indicator if loading or the current content otherwise.